pom.xml

```diff
@@ -161,6 +161,7 @@ under the License.
         <configuration>
           <excludes combine.children="append">
             <exclude>src/it/reproducible/src/main/resources/**/*.txt</exclude>
+            <exclude>**/*.md</exclude>
```

Comment:
please only exclude md in .github directory.
We can use md eg in site

by the way is strange ... eg in maven-invoker-plugin we have template but I don't see excludes ...

Reply:
I'm not sure either but haven't really checked why this behavior.